src/onegov/feriennet/models/notification_template.py

```diff
@@ -145,9 +155,16 @@ def render(self, text: str) -> Markup:
         if len(paragraphs) <= 1:
             result = text
         else:
-            result = Markup('\n').join(as_paragraphs(text))
+            result = '\n'.join(paragraphs)
 
-        return self.expand_storage_links(result)
+        result = self.expand_storage_links(Markup(result))
+
+        return clean(
```

Comment:
I thought about using `linkify` from [core/utils.py](https://github.com/OneGov/onegov-cloud/blob/master/src/onegov/core/utils.py#L406) but it does not allow <p> and <br>. I could easily add them but not sure if its a good idea.